backend/package.json

```
@@ -7,7 +7,7 @@
     "start": "NODE_ENV=production node ./build/index.js",
     "dev": "NODE_ENV=development nodemon index.ts",
     "build": "npm install && npx tsc",
-    "test": "NODE_ENV=test node --test-concurrency=1 --test --require ts-node/register ./tests/auth_api.test.ts",
+    "test": "NODE_ENV=test node --test-concurrency=1 --test --require ts-node/register ./tests/*.test.ts",
```

Test cases are not necessarily always in `tests` folder. We can instead select all `.test.ts` file in this directory.

```suggestion
    "test": "NODE_ENV=test node --test-concurrency=1 --test --require ts-node/register **/*.test.ts",
```